ssh/src/main/java/de/eidottermihi/rpicheck/ssh/impl/RaspiQuery.java

```diff
@@ -75,6 +75,9 @@
     private static final String DISK_USAGE_CMD = "LC_ALL=C df -h";
     private static final String DF_COMMAND_HEADER_START = "Filesystem";
     private static final String DISTRIBUTION_CMD = "cat /etc/*-release | grep PRETTY_NAME";
+    private static final String KERNEL_CMD = "cat /proc/version|cut -f 3 -d \" \"";
+    private static final String ARCHITECTURE_CMD = "(uname -m;echo \" \"; getconf LONG_BIT; echo \"bit\")|tr -d '\\n'";
+    private static final String MODEL_CMD = "cat /proc/cpuinfo |grep 'Model'|cut -f 2 -d \":\"";
```

Comment:
Would'nt it be better to use `cat /sys/firmware/devicetree/base/model` here? According to [Reddit](https://www.reddit.com/r/raspberry_pi/comments/diu58j/simple_command_to_find_model_of_raspberry_pi/?utm_source=share&utm_medium=web2x&context=3) this would simply print Raspberry Pi model and revision.

Reply:
Yes, that's a much better solution.